config/google_tag.container.default.yml

```
@@ -28,10 +32,36 @@ path_list: |-
   /node/*/delete
   /user/*/edit*
   /user/*/cancel*
+  /group/add*
+  /group/*/edit
```

```suggestion
  /group/*/edit*
```

I tried to hit the URL directly then had to login with HID first, which resulted in a querystring when I finally landed on the page.